src/jdk.compiler/share/man/javac.1

```diff
@@ -631,11 +631,6 @@ As applicable, see the descriptions in \f[B]\f[BC]\-\-release\f[B]\f[R],
 details.
 .RE
 .TP
-.B \f[CB]\-\-doclint\-format\f[R] [\f[CB]html4\f[R]|\f[CB]html5\f[R]]
-Specifies the format for documentation comments.
-.RS
-.RE
-.TP
```

Comment:
In general, you should not edit files like this; these files are generated from upstream Markdown files.

Reply:
Ok.  I will revert this with the next commit.